ledger/store/trackerdb/sqlitedriver/sql.go

```diff
@@ -379,7 +383,7 @@ func (qs *accountsDbQueries) LookupResources(addr basics.Address, aidx basics.Cr
 		if err == nil {
 			data.Aidx = aidx
 			if len(buf) > 0 && rowid.Valid {
-				data.Addrid = rowid.Int64
+				data.AcctRef = sqlRowRef{rowid.Int64}
```

Comment:
Just leaving the SQL NullInt64 implementation here, since this made me curious about what it looked like:

```
// NullInt64 represents an int64 that may be null.
// NullInt64 implements the Scanner interface so
// it can be used as a scan destination, similar to NullString.
type NullInt64 struct {
  Int64 int64
  Valid bool // Valid is true if Int64 is not NULL
}
```